packages/client/wallets/smart-wallet/src/SmartWalletSDK.ts

```diff
@@ -44,10 +46,11 @@ export class SmartWalletSDK {
             new EOACreationStrategy(),
             new PasskeyCreationStrategy(crossmintService.getPasskeyServerUrl(), clientApiKey)
         );
+        const accountCache = new AccountConfigCache(`smart-wallet-${SDK_VERSION}`);
```

Comment:
Include the SDK version to invalid cache entries for older versions of the sdk in case there's been updates to how the data is stored.